### DISCUSSION TITLE
If there is any options for preview sidebar?

### DISCUSSION THREAD

**Discussion by hrynevychroman** [TaTo30/vue-pdf#122] [Q&A]
How can I access the custom preview sidebar with this package? Maybe someone achieved this 🤔 
<img width="1728" alt="image" src="https://github.com/TaTo30/vue-pdf/assets/82209198/7dd50b75-950f-499c-ad63-6686de916060">

**Comment by TaTo30** [comment]
This code works for me:

```vue
<div class="column items-center" v-for="i in numPages" :key="i">
     <div
         :class="{
              'pdf-container-selected': currentPage === i,
              'pdf-container': currentPage !== i,
         }"
     >
            <VuePDF :pdf="src" :page="i" fit-parent />
     </div>
     <div class="q-mt-xs">{{ i }}</div>
</div>
```

And this the result:
![image](https://github.com/TaTo30/vue-pdf/assets/57086025/be7e0d81-da8a-4428-a88a-adaeb1a8212c)


`numPages` is gotten from usePDF.
